Kaos-Frontend-Coding-Standards.md

```diff
@@ -159,6 +159,24 @@ At Sonatype we value the stability and maintainability of the code base while st
 * For directives, we use an element restriction when "creating a component that is in control of its template" and an attribute restriction when "decorating an existing element with new functionality".
   * This is inline with the Angular documentation as seen in ["When should I use an attribute versus an element?"](https://docs.angularjs.org/guide/directive)
 
+* We place dependent controllers and links below the directive function
+```javascript
+  function Foo(...) {
+    return {
+      controller: FooController,
+      link: FooLink
+    };
+
+    function FooLink(...) {...}
```

Comment:
I think the link function can really go wherever. I know you had the use case that you wanted the directives injectables which is fine, but if that isn't necessary I think it can go wherever.